### PR TITLE
Fix annotations parsing

### PR DESCRIPTION
**Pull request by Intellium**
This should fix annotation parsing on the service, so that multiple annotations can be specified

### CODE REVIEW

**Comment by itzg** [comment]
Can you provide an example of an error that is reported without the change?

**Comment by Intellium** [comment]
sure.

`Error: YAML parse error on minecraft/charts/minecraft-bedrock/templates/minecraft-svc.yaml: error converting YAML to JSON: yaml: line 11: did not find expected key`

This is with 2 annotations set, like:

```yaml
serviceAnnotations: 
    external-dns.alpha.kubernetes.io/hostname: minecraft.example.com
    my.custom/type: private
```

It seems to render the service template with wrong indentation.
Example:

```yaml
annotations:
        external-dns.alpha.kubernetes.io/hostname: minecraft.example.com
    my.custom/type: private
```

File: charts/minecraft-bedrock/templates/minecraft-svc.yaml [hunk 1]
```diff
@@ -7,8 +7,12 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  {{- if .Values.serviceAnnotations }}
   annotations:
-    {{ toYaml .Values.serviceAnnotations | indent 4 }}
+    {{- range $key, $value := .Values.serviceAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```
Comment (itzg) [inline charts/minecraft-bedrock/templates/minecraft-svc.yaml:14]: I believe the other way to fix the extra indentation was to apply chomping to the left

```
    {{- toYaml .Values.serviceAnnotations | indent 4 }}
```

...however, your change is more consistent with similar sections in the templates, so the current change is best.

**Review by itzg — COMMENTED** [review]
Just need the chart version bumped and then this is ready for a merge.

**Comment by itzg** [comment]
...and thanks for providing the additional information.

**Comment by Intellium** [comment]
Bumped the version

**Review by zetneteork — COMMENTED** [review]
LGTM

**Pull request merged**